engine/crates/composition/tests/composition/tag_directive_basic/api.graphql

```diff
@@ -0,0 +1,31 @@
+enum AppleVariety @tag(name: "appleEnum") {
+    FUJI
+    GRANNY_SMITH
+    HONEYCRISP @tag(name: "appleEnumValue")
+}
+
+type Apple @tag(name: "appleType") {
+    id: ID!
+    texture: Texture
+    variety: AppleVariety
+}
+
+type Orange @tag(name: "orangeType") {
```

Comment:
Hmm I think `@tag` shouldn't be in the public API. right?

Reply:
No idea for `@tag` to honest. I don't think we should treat it as a public API. The public API is introspection. This is the schema we show to the maintainer/developers of the project. So they need to be aware of the business logic that affects runtime behavior IMHO. We should only strip out the federation aspects (keys, provides, inaccessible, override, etc.)